### PR TITLE
Feature/pelagos 3558 add additional properties to research group

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Comment by praneethpr** [comment]
Travis builds which execute codesniffer phpcs are skipping few syntax rules which are enforced on gcs

File: src/Pelagos/Entity/ResearchGroup.php [hunk 1]
```diff
@@ -86,6 +86,21 @@ class ResearchGroup extends Entity
      */
     protected $name;
 
+    /**
+     * Name of a research group.
+     *
+     * @var string $name
```
Comment (mickel1138) [inline src/Pelagos/Entity/ResearchGroup.php]: set to correct property name.

File: src/Pelagos/Entity/ResearchGroup.php [hunk 2]
```diff
@@ -86,6 +86,21 @@ class ResearchGroup extends Entity
...
+     *
+     * @access protected
+     *
+     * @ORM\Column(type="citext", unique=true, nullable=true)
```
Comment (mickel1138) [inline src/Pelagos/Entity/ResearchGroup.php]: Citext is case insensitive text, also uniqueness must be enforced.

File: src/Pelagos/Entity/ResearchGroup.php (hunk 1 above)
Comment (praneethpr) [reply src/Pelagos/Entity/ResearchGroup.php]: 👍

File: src/Pelagos/Entity/ResearchGroup.php (hunk 2 above)
Comment (fightingtexasaggie) [reply src/Pelagos/Entity/ResearchGroup.php]: I tested and it does recognize "oyster" and "Oyster" as being the same, which I think is the right behavior for uniqueness. (the violation just needs to be handled)

Comment (praneethpr) [reply src/Pelagos/Entity/ResearchGroup.php]: Added the unique constraint validator

**Pull request merged**